Scripts/DowloadPortalItemData.swift

```diff
@@ -218,7 +208,7 @@ let portalItems: [PortalItem] = {
         // Omit the decoding errors from samples that don't have dependencies.
         let sampleDependencies = sampleJSONs
             .compactMap { try? parseJSON(at: $0) }
-        return sampleDependencies.flatMap(\.offlineData)
+        return Set(sampleDependencies.flatMap(\.offlineData))
```

Comment:
This is a great catch. We used to repeatedly download files…

Reply:
More important than this is the guard statement inside the `forEach` loop. With that, filtering out duplicates isn't as important. Still good to do, but not as big of a deal.